.github/workflows/build-linux-arm64-installer.yml

```diff
@@ -114,32 +114,45 @@ jobs:
 
     - name: Create Checksums
-      if: startsWith(github.ref, 'refs/tags/')
+      if: startsWith(github.ref, 'refs/tags/') || github.ref == 'refs/heads/main`
       env:
         CHIA_INSTALLER_VERSION: ${{ steps.version_number.outputs.CHIA_INSTALLER_VERSION }}
       run: |
         ls ${{ github.workspace }}/build_scripts/final_installer/
-        sha256sum ${{ github.workspace }}/build_scripts/final_installer/chia-blockchain_${{ steps.version_number.outputs.CHIA_INSTALLER_VERSION }}_arm64.deb > ${{ github.workspace }}/build_scripts/final_installer/chia-blockchain_${{ steps.version_number.outputs.CHIA_INSTALLER_VERSION }}_arm64.deb.sha256
+        sha256sum ${{ github.workspace }}/build_scripts/final_installer/chia-blockchain_${{ CHIA_INSTALLER_VERSION }}_arm64.deb > ${{ github.workspace }}/build_scripts/final_installer/chia-blockchain_${{ CHIA_INSTALLER_VERSION }}_arm64.deb.sha256
         ls ${{ github.workspace }}/build_scripts/final_installer/
 
     - name: Install py3createtorrent
-      if: startsWith(github.ref, 'refs/tags/')
+      if: startsWith(github.ref, 'refs/tags/') || github.ref == 'refs/heads/main'
       run: |
         pip3 install py3createtorrent
 
     - name: Create torrent
-      if: startsWith(github.ref, 'refs/tags/')
+      if: startsWith(github.ref, 'refs/tags/') || github.ref == 'refs/heads/main'
+      env:
+        CHIA_INSTALLER_VERSION: ${{ steps.version_number.outputs.CHIA_INSTALLER_VERSION }}
       run: |
-        py3createtorrent -f -t udp://tracker.opentrackr.org:1337/announce ${{ github.workspace }}/build_scripts/final_installer/chia-blockchain_${{ steps.version_number.outputs.CHIA_INSTALLER_VERSION }}_arm64.deb -o ${{ github.workspace }}/build_scripts/final_installer/chia-blockchain_${{ steps.version_number.outputs.CHIA_INSTALLER_VERSION }}_arm64.deb.torrent --webseed https://download-chia-net.s3.us-west-2.amazonaws.com/install/chia-blockchain_${{ steps.version_number.outputs.CHIA_INSTALLER_VERSION }}_arm64.deb
+        py3createtorrent -f -t udp://tracker.opentrackr.org:1337/announce ${{ github.workspace }}/build_scripts/final_installer/chia-blockchain_${{ CHIA_INSTALLER_VERSION }}_arm64.deb -o ${{ github.workspace }}/build_scripts/final_installer/chia-blockchain_${{ steps.version_number.outputs.CHIA_INSTALLER_VERSION }}_arm64.deb.torrent --webseed https://download-chia-net.s3.us-west-2.amazonaws.com/install/chia-blockchain_${{ CHIA_INSTALLER_VERSION }}_arm64.deb
         ls ${{ github.workspace }}/build_scripts/final_installer/
 
+    - name: Upload Beta Installer
+      if: steps.check_secrets.outputs.HAS_SECRET && github.ref == 'refs/heads/main'
+      env:
+        CHIA_INSTALLER_VERSION: ${{ steps.version_number.outputs.CHIA_INSTALLER_VERSION }}
+      run: |
+        aws s3 cp ${{ github.workspace }}/build_scripts/final_installer/chia-blockchain_${{ CHIA_INSTALLER_VERSION }}_arm64.deb s3://download-chia-net/beta/chia-blockchain_${{ CHIA_INSTALLER_VERSION }}_arm64_latest_beta.deb
```

Comment:
@nirajpathak13 we should probably leave the version off the s3 destination name, so that we can just have a generic filename for "this is the latest installer from main"